### PR TITLE
[Compile Time Constant Extraction] Extract Runtime Metadata Attributes

### PR DESCRIPTION
**Pull request by artemcm**
Part of rdar://104192094

### CODE REVIEW

**Comment by artemcm** [comment]
@swift-ci test

File: lib/AST/Decl.cpp [hunk 1]
```diff
@@ -6814,6 +6814,14 @@ bool VarDecl::hasAttachedPropertyWrapper() const {
   return false;
 }
 
+/// Whether this property has any attached runtime metadata attributes.
+bool VarDecl::hasRuntimeMetadataAttributes() const {
+  if (!getRuntimeDiscoverableAttrs().empty())
```
Comment (xedin) [inline lib/AST/Decl.cpp:6819]: Nit: `return !getRuntimeDiscoverableAttrs().empty();` should do it :)

**Review by xedin — APPROVED** [review]

File: lib/AST/Decl.cpp (hunk 1 above)
Comment (artemcm) [reply lib/AST/Decl.cpp]: 😬 Thanks!

**Comment by artemcm** [comment]
@swift-ci test and merge

**Comment by artemcm** [comment]
@swift-ci test macOS Platform

**Comment by artemcm** [comment]
@swift-ci test macOS Platform

**Pull request merged**